server/package.json

```diff
@@ -10,6 +10,7 @@
     "dev": "nodemon ./bin/www"
   },
   "dependencies": {
+    "@sendgrid/mail": "^7.6.0",
```

Comment:
I think this dependency was added on this PR: https://github.com/hatchways/team-scallops/pull/104/files.

Reply:
is pr #104 merged in to develop?

Reply:
Not yet, I just wanted to make the clarification since I made the PR for the issue you linked 😅 But I supposed you may need this dependency for another feature but if not, it will be added there. 👍🏽

Reply:
No worries, we can merge it in as it isn't going to break anything. Just wanted to ensure development had mirrored the change I needed to make to staging. 

If you may check something out of curiosity, do you have package.json in your .gitignore file? I don't think so, but I can't put my finger on why this was excluded or required in staging. It won't break anything if we include it so It's not a horrible idea to merge it into development but would be unnecessary unless it appears as an issue. We can close this PR for now.